bigbluebutton-config/bin/bbb-conf

```diff
@@ -693,6 +701,43 @@ if [[ $SECRET ]]; then
     echo
 fi
 
+#
+# Set Port Range
+#
+
+if [[ $PORT_RANGE ]]; then
+    if [[ "$PORT_RANGE" =~ ^([[:digit:]]+)-([[:digit:]]+)$ ]]; then
+        START_PORT=${BASH_REMATCH[1]}
+        END_PORT=${BASH_REMATCH[2]}
+
+        need_root
+
+        xmlstarlet edit --inplace --update '/configuration/settings/param[@name="rtp-start-port"]/@value' --value $START_PORT $FREESWITCH_SWITCH_CONF
+        xmlstarlet edit --inplace --update '/configuration/settings/param[@name="rtp-end-port"]/@value' --value $END_PORT $FREESWITCH_SWITCH_CONF
+
+        sed -i "s/minPort=.*/minPort=$START_PORT/" /etc/kurento/modules/kurento/BaseRtpEndpoint.conf.ini
+        sed -i "s/maxPort=.*/maxPort=$END_PORT/" /etc/kurento/modules/kurento/BaseRtpEndpoint.conf.ini
+
+        yq w -i /usr/local/bigbluebutton/bbb-webrtc-sfu/config/default.yml mediasoup.worker.rtcMinPort $START_PORT
```

Comment:
these settings should be set in `/etc/bigbluebutton/bbb-webrtc-sfu/production.yml` instead of modifying the file shipped with the packages. Changes in `/etc/bigbluebutton` will survive a package upgrade, changes in `/usr/local/bigbluebutton/bbb-webrtc-sfu` wont.

Reply:
Agreed.  I pushed an update.

The FreeSWITCH and Kurento settings can't be changed in `/etc/bigbluebutton` (as far as I know), but that's an issue for a different PR.